kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -2381,6 +2455,114 @@ public void testFindCaseTasksAssignedAsStakeholderPagination() {
         assertPagination((page, pageSize) -> caseClient.findCaseTasksAssignedAsStakeholder(caseId, USER_YODA, page, pageSize), 2);
     }
 
+    private void assertTaskListAsPotentialOwner(String caseId, String caseOwner, String potOwner, boolean isInHR) {
+        List<TaskSummary> tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, caseOwner, 0, 10);
+        assertNotNull(tasks);
+        assertEquals(1, tasks.size());
+        assertEquals("Hello1", tasks.get(0).getName());
+        assertEquals(caseOwner, tasks.get(0).getActualOwner());
+        assertEquals(Reserved, Status.valueOf(tasks.get(0).getStatus()));
+
+        Map<String, Object> taskInput = new HashMap<>();
+        taskInput.put("ActorId", potOwner);
```

Comment:
You could define ActorId, GroupId, Comment and BusinessAdministratorId as constants.

Reply:
done